### ISSUE TITLE
required annotation not required when creating edges

### ISSUE THREAD

**Issue by keski** [LiUGraphQL/woo.sh#92]
Field annotations containing a required field are not marked as required in the API. For example, assume that we have the following DB schema:

```graphql
type Human {
  name: String!
  friends(level: Float!): [Human]
}
```

Following the API generation, we get the following API schema:
```graphql
input _InputToAnnotateFriendsEdgeFromHuman {
  level: Float!
}

input _InputToConnectFriendsOfHuman {
  connect: ID
  create: _InputToCreateHuman
  annotations: _InputToAnnotateFriendsEdgeFromHuman # Why is the annotation not required?
}

input _InputToCreateFriendsEdgeFromHuman {
  sourceID: ID!
  targetID: ID!
  annotations: _InputToAnnotateFriendsEdgeFromHuman # Why is the annotation not required?
}

input _InputToCreateHuman {
  name: String!
  friends: [_InputToConnectFriendsOfHuman]
}

# ... more
```
Since the annotation contains at least one field that is required (`level: Float!`) I would expect the annotation field to also be required. Is this interpretation correct?

@hartig @epii-1 @rcapshaw

**Comment by epii-1** [comment]
Seems resonable. I'll correct it.

**Comment by epii-1** [comment]
Should now be corrected with commit bcb8b79
